assets/js/vue/components/FundingCyclesList.vue

```diff
@@ -34,67 +33,80 @@
                 })
+                this.disableResearchGroups = false;
             },
-            selectResearchGroup: function(event) {
-                if (event.target.value) {
-                    this.openResearchGroupLandingPage(event.target.value);
+            researchGroupButton: function() {
+                this.openResearchGroupLandingPage(this.selectedResearchGroup);
+            },
+            openResearchGroupLandingPage: function (researchGroupId) {
+                if (researchGroupId) {
+                    window.open("/research-group/about/" + researchGroupId, '_blank');
                 }
             },
-            populateProjectDirectors: function () {
+            populateFundingCycles: function () {
                 this.fundingCycles.forEach(fundingCycle => {
-                    fundingCycle.researchGroups.forEach(researchGroup => {
-                        researchGroup.projectDirectors.forEach(projectDirector => {
-                            if (this.projectDirectorIds.indexOf(projectDirector.id) === -1) {
-                                this.projectDirectorIds.push(projectDirector.id);
-                                this.makeProjectDirectorOption(researchGroup.id, projectDirector.name);
-                            } else {
-                                this.projectDirectorIds.push(projectDirector.id);
-                                this.makeProjectDirectorOption(
-                                    researchGroup.id,
-                                    projectDirector.name + ' - ' + researchGroup.shortName
-                                );
-                            }
-                        })
+                    this.fundingCycleOptions.push({
+                        value: fundingCycle.id,
+                        text: fundingCycle.name
                     })
                 })
             },
-            openResearchGroupLandingPage: function (researchGroupId) {
-                window.open("/research-group/about/" + researchGroupId, '_blank');
+            projectDirectorButton: function () {
+                this.openResearchGroupLandingPage(this.selectedProjectDirector);
             },
-            makeProjectDirectorOption: function (id, name) {
-                this.projectDirectorsOptions.push({
-                    value: id,
-                    text: name
+            populateProjectDirectors: function () {
+                this.projectDirectors.forEach(projectDirector => {
+                    this.projectDirectorsOptions.push({
+                        value: projectDirector.researchGroupId,
+                        text: projectDirector.name
+                    })
                 })
             }
         },
         created() {
+            this.populateFundingCycles();
             this.populateProjectDirectors();
         },
         watch: {
+            selectedFundingCycle: function () {
+                this.populateResearchGroups(this.selectedFundingCycle);
```

Comment:
Also disable researchgroup (and maybe select placeholder)

Reply:
I didn't quite understand what you meant

Reply:
When you select a different Funding Cycle, or select the placeholder, the research group should be disabled again.